content/documentation/figure-images.md

```diff
@@ -157,7 +157,7 @@ Quire supports video embeds from either YouTube (`media_type: youtube`) or Vimeo
 
 ## Add Tables
 
-Zoomable tables can be embedded in your publication as HTML files. In the `static/image/` directory of your project create a new folder called `tables`. Create an .html file in that folder that follows the structure below:
+In addition inserting tables in your Project as Markdown, can you also embed zoomable tables as HTML files. In the `static/img/` directory of your project create a new folder called `tables`. Create an .html file in that folder that follows the structure below:
```

Comment:
Is "zoomable" the right description for these? Just asking. They do open up in a bigger view, but you can't zoom beyond that. What do you think?

Reply:
Oh, for some reason I thought you could zoom further in on them. I guess it's just a matter of what makes them better/different than Markdown tables. Are they considered more "dynamic" or "interactive"? Or is it really just that they pop out / you can make them bigger?